### PR TITLE
[Symfony UX Map] Add post-install steps for Leaflet and Google, add "ux_map.google_maps.default_map_id" config recipe

### PR DESCRIPTION
**Pull request by Kocal**
| Q             | A
| ------------- | ---
| License       | MIT
| Doc issue/PR  | symfony/symfony-docs#...

Follow https://github.com/symfony/ux/pull/2358, I've also added post-install steps.

Version 2.22 is not released yet, [but 2.19 is](https://packagist.org/packages/symfony/ux-google-map#v2.19.0)
image

### CODE REVIEW

**Review by fabpot — APPROVED** [review]

File: symfony/ux-google-map/2.22/manifest.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        {
+            "file": "config/packages/ux_map.yaml",
+            "position": "after_target",
+            "warn_if_missing": true,
+            "target": "    renderer: '%env(resolve:default::UX_MAP_DSN)%'",
+            "content": "    google_maps:\n        # define the default map id for all maps (https://developers.google.com/maps/documentation/get-map-id)\n        default_map_id: null"
+        }
+    ],
+    "env": {
+        "#1": "Options available at https://github.com/symfony/symfony-ux/blob/2.x/src/Map/src/Bridge/Google/README.md",
```
Comment (ker0x) [inline symfony/ux-google-map/2.22/manifest.json:15]:
```suggestion
        "#1": "Options available at https://github.com/symfony/ux/blob/2.x/src/Map/src/Bridge/Google/README.md",
```

File: symfony/ux-google-map/2.22/post-install.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+  * <fg=blue>Read</> the documentation at <comment>https://github.com/symfony/symfony-ux/blob/2.x/src/Map/src/Bridge/Google/README.md</>
```
Comment (ker0x) [inline symfony/ux-google-map/2.22/post-install.txt:1]:
```suggestion
  * <fg=blue>Read</> the documentation at <comment>https://github.com/symfony/ux/blob/2.x/src/Map/src/Bridge/Google/README.md</>
```

File: symfony/ux-leaflet-map/2.19/post-install.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+  * <fg=blue>Read</> the documentation at <comment>https://github.com/symfony/symfony-ux/blob/2.x/src/Map/src/Bridge/Leaflet/README.md</>
```
Comment (ker0x) [inline symfony/ux-leaflet-map/2.19/post-install.txt:1]:
```suggestion
  * <fg=blue>Read</> the documentation at <comment>https://github.com/symfony/ux/blob/2.x/src/Map/src/Bridge/Leaflet/README.md</>
```

File: symfony/ux-google-map/2.19/post-install.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+  * <fg=blue>Read</> the documentation at <comment>https://github.com/symfony/symfony-ux/blob/2.x/src/Map/src/Bridge/Google/README.md</>
```
Comment (ker0x) [inline symfony/ux-google-map/2.19/post-install.txt:1]:
```suggestion
  * <fg=blue>Read</> the documentation at <comment>https://github.com/symfony/ux/blob/2.x/src/Map/src/Bridge/Google/README.md</>
```

File: symfony/ux-google-map/2.22/manifest.json (hunk 1 above)
Comment (Kocal) [reply symfony/ux-google-map/2.22/manifest.json]: Thanks Romain!

**Review by fabpot — APPROVED** [review]

**Pull request merged**